### ISSUE TITLE
v5.1.0-0 Error: bundling failed: Error: Unable to resolve module `./action` from `node_modules/typesafe-actions/dist/typesafe-actions.umd.production.js`

### ISSUE THREAD

**Issue by jstheoriginal** [piotrwitek/typesafe-actions#202]
## Description
React Native reports this error: 

bundling failed: Error: Unable to resolve module `./action` from `node_modules/typesafe-actions/dist/typesafe-actions.umd.production.js`

I've confirmed that the file it's trying to get isn't there (only the action.d.ts file is there)

## Expected behavior
Should bundle properly onto device.

## Suggested solution(s)
I'm guessing it's a distribution issue since the files exist in the repo.

## Project Dependencies
- Typesafe-Actions Version: 5.1.0-0
- TypeScript Version: 3.6.4
- tsconfig.json:
```
{
  "compilerOptions": {
    "allowJs": true,
    "checkJs": false,
    "allowSyntheticDefaultImports": true,
    "esModuleInterop": true,
    "isolatedModules": true,
    "jsx": "react",
    "lib": ["es2017"],
    "module": "commonjs",
    "moduleResolution": "node",
    "noEmit": true,
    "incremental": false, // must be false if noEmit is true
    "strict": true,
    "target": "esnext",
    "importHelpers": true,
    "skipLibCheck": true,
    "resolveJsonModule": true,
    "noEmitOnError": false,
    "noFallthroughCasesInSwitch": true,
    "noUnusedLocals": true,
    "pretty": true,
    "removeComments": true,
    "sourceMap": true,
    "forceConsistentCasingInFileNames": true,
    "experimentalDecorators": true,
    "baseUrl": ".",
    "paths": {
      "*": [
        "src/*"
      ]
    }
  },
  "include": [
    "src",
    "@types",
    "cloudkit-js",
    "babel.config.js"
  ]
}
```

## Environment (optional)
- Browser and Version: n/a
- OS: Android
- Node Version: v10.16.3
- Package Manager and Version: react-native-cli@2.0.1

**Comment by piotrwitek** [comment]
Hey @jstheoriginal 
I updated rollup, that's probably some configuration issue, thanks for report I'll look into it ASAP

**Comment by piotrwitek** [comment]
Released `typesafe-actions@5.1.0-1` with the fix 🎉

**Comment by jstheoriginal** [comment]
Thanks for the quick fix!
